.travis.yml

```diff
@@ -36,8 +36,9 @@ notifications:
   email: false
 
 script:
-  - mvn -q test -Pspark-1.6
   - mvn -q test -Pspark-2.0
+  # test the spark-1.6 module only in this second run
+  - mvn -q test -Pspark-1.6 -Dtest=org.apache.spark.*
```

Comment:
I think it should be tested both on spark 1.6 and spark 2.0.
Before committing this change, test was run successfully. Is this change required?